test/resources/invalid.txt

```diff
@@ -0,0 +1,187 @@
+                            <SuiteIdentifier>Sir Humphrey Suite</SuiteIdentifier>
+                            <FloorIdentifier>Second</FloorIdentifier>
+                            <DistrictName>Westminster</DistrictName>
+                            <POB>48</POB>
+                            <PostCode>SW1A 4GG</PostCode>
+                            <City>London</City>
+                            <Country>GB</Country>
+                        </Address>
+                        <EmailAddress>test@digital.hmrc.gov.uk</EmailAddress>
+                        <ResCountryCode>VU</ResCountryCode>
+                    </Individual>
+                </ID>
+                <Capacity>DAC61102</Capacity>
+                <NationalExemption>
+                    <Exemption>true</Exemption>
+                    <CountryExemptions>
+                        <CountryExemption>VU</CountryExemption>
+                    </CountryExemptions>
+                </NationalExemption>
+            </Intermediary>
+        </Intermediaries>
+        <AffectedPersons>
+            <AffectedPerson>
+                <AffectedPersonID>
+                    <Individual>
+                        <IndividualName>
+                            <FirstName>Palmerston</FirstName>
+                            <LastName>C</LastName>
+                            <Suffix>(Cat)</Suffix>
+                        </IndividualName>
+                        <BirthDate>2012-01-14</BirthDate>
+                        <BirthPlace>Hexham</BirthPlace>
+                        <TIN issuedBy="GB">AB000000D</TIN>
+                        <Address>
+                            <Street>King Charles Street</Street>
+                            <BuildingIdentifier>No 10</BuildingIdentifier>
+                            <SuiteIdentifier>Lord Palmerston Suite</SuiteIdentifier>
+                            <FloorIdentifier>Second</FloorIdentifier>
+                            <DistrictName>Westminster</DistrictName>
+                            <POB>48</POB>
+                            <PostCode>SW1A 4GG</PostCode>
+                            <City>London</City>
+                            <Country>GB</Country>
+                        </Address>
+                        <EmailAddress>test@digital.hmrc.gov.uk</EmailAddress>
+                        <ResCountryCode>VU</ResCountryCode>
+                    </Individual>
+                </AffectedPersonID>
+            </AffectedPerson>
+        </AffectedPersons>
+        <DisclosureInformation>
+            <ImplementingDate>2020-01-14</ImplementingDate>
+            <Reason>DAC6704</Reason>
+            <Summary>
+                <Disclosure_Name>Catnip Advice</Disclosure_Name>
+                <Disclosure_Description>I will give advice on expensing catnip for money</Disclosure_Description>
+            </Summary>
+            <NationalProvision>I don't know what this is as I am just a cat</NationalProvision>
```

Comment:
If this project is to be open sourced in the future they may be a problem with the humorous test data